apollo-client/src/main/java/com/ctrip/framework/apollo/util/ConfigUtil.java

```diff
@@ -56,7 +56,8 @@
   private boolean autoUpdateInjectedSpringProperties = true;
   private final RateLimiter warnLogRateLimiter;
   private boolean propertiesOrdered = false;
```

Comment:
why did you change the default value of  `propertyNamesCacheEnabled`? but did not change default value of `propertiesOrdered`?

Reply:
@zouyx I am not chaning the default value. I just extract a method for `propertyNamesCacheEnabled`

Reply:
The boolean variable's default value is false, so setting false or not is basically the same thing.
But this change does bring some confusion, so I would suggest we don't change it.